hack/generate/changelog/util/fragment_test.go

```diff
@@ -370,3 +373,51 @@ func TestGitPullRequestNumberGetter_parsePRNumber(t *testing.T) {
 		})
 	}
 }
+
+func TestGitPullRequestNumberGetter_getCommitMessage(t *testing.T) {
```

Comment:
Can you re-write this test using [ginkgo](https://onsi.github.io/ginkgo/)/[gomega](https://onsi.github.io/gomega)? We're trying to do this in the rest of the SDK.

Reply:
Usually, I agree, however, since it is with tests using the golang lib it means to change all tests and it is just used for the release. So, would be really valid/worth in this case? I am ok with the fix without to move all tests for ginkgo/gomega but it is up to you folks. WDYT ? @joelanford

Reply:
Eh that's fine, it would be easier to read though.